### ISSUE TITLE
Unmap cmd + left / cmd + right arrow behavior

### ISSUE THREAD

**Issue by oliverswitzer** [philc/vimium#3873]
How does one unmap the default behavior to go back or forward in the browser when pressing cmd + <left> arrow or cmd + <right> arrow respectively? I have so far tried:

unmap <left>
unmap cmd + <left>

And neither appears to work

The reason for wanting to unmap is because it prevents me from doing proper line selection (cmd + left would normally highlight the text to the left of the cursor)

An alternative solution is to only allow this command to send you back or forward in history if you're *not* focused in an text input

**Comment by gdh1995** [comment]
`cmd+left/right` seems not in the default list of key mappings in Vimium.

**Comment by oliverswitzer** [comment]
Hmm, odd. I’ve tried uninstalling Vimium and the behavior (came + left
arrow navigates backward in history and vice versa for right arrow)
disappears for me when I do. It reappears when I reinstall Vimium… can you
confirm that this behavior does not appear for you on a fresh install?

**Comment by gdh1995** [comment]
Have you modified "custom key mappings" on Vimium Options?

**Comment by oliverswitzer** [comment]
@gdh1995 nope, only trying to "unmap" the arrow key functionality in this section of Vimium options like I mentioned in the issue.
